CIP-0067/registry.schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/cardano-foundation/CIPs/blob/master/CIP-?",
+  "type": "array",
+  "title": "Asset Name Label Registry",
+  "description": "JSON schema for asset name label registry",
+  "default": [],
+  "examples": [
+    [
+      {
+        "asset_name_label": 721,
+        "class": "NFT",
+        "description": "CIP-0025 - NFT Metadata Standard"
+      }
+    ]
+  ],
+  "additionalItems": false,
+  "items": {
+    "$id": "#/items",
+    "anyOf": [
+      {
+        "$id": "#/items/anyOf/0",
+        "type": "object",
+        "title": "The first anyOf schema",
+        "description": "An entry in the asset name label registry",
+        "default": {},
+        "examples": [
+          {
+            "asset_name_label": 721,
+            "class": "NFT",
+            "description": "CIP-0025 - NFT Metadata Standard"
+          }
+        ],
+        "required": ["asset_name_label", "class", "description"],
+        "properties": {
+          "asset_name_label": {
+            "$id": "#/items/anyOf/0/properties/asset_name_label",
+            "type": "integer",
+            "title": "The asset_name_label number",
+            "default": 0,
+            "examples": [1967]
+          },
+          "class": {
+            "$id": "#/items/anyOf/0/properties/class",
+            "type": "string",
+            "title": "The asset class",
```

Comment:
This is nowhere mentioned in the specification. If this can only be `FT` or `NFT`, I suggest to add an extra validation to this subschema: 

```
"enum": [ "FT", "NFT" ]
```

Reply:
Logically we can only have `FT` and `NFT`, but I think there can be more categories in the end. E.g. Semi fungible tokens (`SFT`), etc. Whatever establishes in the future.